app/code/Magento/Captcha/view/adminhtml/templates/default.phtml

```diff
@@ -7,24 +7,25 @@
 /** @var \Magento\Captcha\Block\Captcha\DefaultCaptcha $block */
 /** @var \Magento\Captcha\Model\DefaultModel $captcha */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
+/** @var \Magento\Framework\Escaper $escaper */
```

Comment:
Could you combine it to a single phpdoc block?

Reply:
fixed